docs/skopeo-docker-list-tags.1.md

```diff
@@ -0,0 +1,72 @@
+% skopeo-docker-list-tags(1)
+
+## NAME
+skopeo\-docker\-list\-tags - Return a list of tags from _repository-name_ in a registry
+
+## SYNOPSIS
+**skopeo docker-list-tags** [**--config**] _repository-name_
+
+Return a list of tags from _repository-name_ in a registry
+
+  _repository-name_ name of repository to retrieve tag listing from
+
+  **--config** output configuration in OCI format, default is to format in JSON
+
+  _image-name_ name of image to retrieve configuration for
+
+  **--config** **--raw** output configuration in raw format
```

Comment:
These options don’t exist.

(Note to self: I haven’t read the man page word by word yet.)